frontend/src/views/web/challenge/my-submission.html

```diff
@@ -35,7 +35,7 @@
                                     <tr>
                                         <th data-field="team">Team Name</th>
                                         <th data-field="status">Status</th>
-                                        <th data-field="status">Execution Time (In sec.)</th>
+                                        <th style="text-align:center" data-field="status">Execution Time (sec.)</th>
```

Comment:
I think we already have one... check in base.scss for center aligned text. It might be "text-center". Just check, if not present then add it.